plaidml2/core/ffi.cc

```diff
@@ -2,11 +2,20 @@
 
 #include "plaidml2/core/ffi.h"
 
+#include <stdlib.h>
```

Comment:
Let's use `<cstdlib>` instead (it's the same, but at least it's consistent with the other includes.